### ISSUE TITLE
Usar NodeJS

### ISSUE THREAD

**Issue by RodolfoSilva** [SnappyDroid/SnappyDroid_V1#6]
Poderíamos utilizar o Johnny Five para controlar o Arduíno, utilizando o protocolo Firmata, e todo o resto por conta do NodeJS. O que acham?

**Comment by LuccaPrado** [comment]
Acho bem interessante, isso iria deixar o projeto acessível a mais pessoas
por sua facilidade

**Comment by RodolfoSilva** [comment]
Sim, e poderíamos tornar ainda mais poderoso o Snappy... E não ficaríamos limitados aos 32KB de memória do ATMEGA.

**Comment by sinesiobittencourt** [comment]
Pessoal, show mesmo. 

Vamos mandar bala então ;) 

Estou trabalhando no novo controlador de voz dele e em breve código aqui :)
